srv/npm/npm-srv.js

```diff
@@ -7,7 +7,7 @@ module.exports = npm
 
 // TODO: move to model
 module.exports.defaults = {
-  registry: 'http://registry.npmjs.org/',
+  registry: 'https://replicate.npmjs.com/',
```

Comment:
The `'https://replicate.npmjs.com/'` url is repeated in three places across the codebase. Is there a specific pattern you would like us to follow in order to DRY it up, - or am I free to make a judgment call on this one?